src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,61 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int AGE_MIN = 18;
+    private static final int MIN_PASSWORD = 6;
+    private static final String DO_NOT_USE_NAME = "Maksym";
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        validateUser(user);
+        validateAge(user);
+        validateLogin(user);
+        validatePassword(user);
+        validateName(user);
```

Comment:
We do not need all these private methods
![image](https://user-images.githubusercontent.com/20326858/205263034-2763b7cd-fe36-4448-9576-d1571256268f.png)